docs/examples/performance-ratio-nrel.py

```diff
@@ -0,0 +1,74 @@
+"""
+Calculate Performance Ratio (NREL)
+==================================
+
+Calculate the NREL Performance Ratio for a system.
+"""
+
+# %%
+# Identifying the NREL performance ratio (PR) is a simple way to communicate
+# how the system is performing, in terms of both system quality and weather.
+# The PR is the ratio of the actual electricity generated to the electricity
+# that would be generated if the plant consistently converted sunlight to
+# electricty at the level of the DC nameplate rating. In this example, we
```

Comment:
Technically this is describing the original (non-temperature-adjusted) PR.  Here's a suggested revision of the first two sentences:

> When evaluating PV system performance it is often desirable to distinguish uncontrollable effects like weather variation from controllable effects like soiling and hardware issues.  The NREL Performance Ratio (or "Weather-Corrected Performance Ratio") is a unitless metric that normalizes system output for variation in irradiance and temperature, making it insensitive to uncontrollable weather variation and more reflective of system health.

Reply:
Changed in https://github.com/pvlib/pvanalytics/pull/153/commits/cdb667cf012a24f9b2f8c9d06fee422c4cd727ef